ui/src/api/files/utils.ts

```diff
@@ -72,6 +80,8 @@ export const fileTypeToIcon = (type: BaseFileSchemaType["type"]) => {
       return Workflow;
     case "consumer":
       return Users;
+    case "page":
+      return PanelTop;
```

Comment:
This icon is used for the breadcrumb and the file browser, and it is currently different from the one in the "Create new / New Page" menu. I think it would be nice if this would be the same icon. I leave you the decision on which Icon fits best. I can't decide which one I like better right now :D